util/rbac/rbac.go

```diff
@@ -97,6 +98,26 @@ func (e *Enforcer) EnforceErr(rvals ...interface{}) error {
 			for i, rval := range rvals[1:] {
 				rvalsStrs[i] = fmt.Sprintf("%s", rval)
 			}
+			switch s := rvals[0].(type) {
+			case jwt.Claims:
+				claims, err := jwtutil.MapClaims(s)
+				if err != nil {
+					break
+				}
+				sub := jwtutil.GetField(claims, "sub")
+				if sub != "" {
+					rvalsStrs = append(rvalsStrs, fmt.Sprintf("sub: %s", sub))
+				}
+				iatField, ok := claims["iat"]
+				if !ok {
+					break
+				}
+				iat := iatField.(float64)
```

Comment:
Looks like ok is missing:
```suggestion
				iat, ok := iatField.(float64)
```